go/ct/spc/specification.go

```diff
@@ -1387,6 +1414,7 @@ func getAllRules() []Rule {
 		pushes:    1,
 		conditions: []Condition{
 			Eq(ReadOnly(), true),
+			RevisionBounds(MinRevision, R11_Paris),
```

Comment:
This is a temporary change, right? Until they are updated to match Shanghai?
If so, maybe add a TODO indicating that this needs to be cleaned up.

Reply:
Right, we really need to be sure that this code goes away whoever merges first.

Catch 22 scenario: 
- if I commit without bumping up newest supported: my code panics with the new check or it did something wrong without.
- If I do upgrade, the newest supported rev:  Code Size change is not yet there and ct starts failing for create. 

We flirted with the idea of adding a RevisionBounds mechanism so that if the revision is future to the currently supported one, the ct would ignore it. Nevertheless, this would require some refactoring between conditions and generators, and we discussed offline that it does not worth the effort.